### ISSUE TITLE
[Designer]Bug 33489363: [FoodOrder.Json]: Screen reader is not announcing the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Adaptive Card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7626]
### Problem Description

[Bug 33489363](https://microsoft.visualstudio.com/OS/_workitems/edit/33489363): [FoodOrder.Json]: Screen reader is not announcing the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Adaptive Card.

Test Environment:

Application: Adaptive Cards
OS version: Dev (OS Build 21387.1)

Application Version: 2.7.2105.11001

Screen Reader: Narrator

Repro steps:

1. Launch the Adaptive Card application and narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to FoodOrder.Json button by using Tab key and activate it.

4. FoodOrder adaptive card will be appeared on right sight of the pane.

5. Navigate to FoodOrder card by using Tab key.

6. Navigate to "Steak", "Chicken" and "Tofu" buttons with Tab key and observe the issue.

Actual Result:
Screen reader is not announcing the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Adaptive Card. Screen Reader is only announcing 'Steak button'.

Expected Result:
Screen reader should announce the 'Expand/Collapse' state of "Steak", "Chicken" and "Tofu" buttons under 'FoodOrder.json. Adaptive Card. Screen reader should announce as, "Steak button Expanded/Collapsed".

User Experience:
User depending on screen reader might get affected if screen reader does not pass State information about the control. Screen reader user will not able to know the property of that control and will face problem while interacting with it.

Observation: Issue also repro with NVDA(2020.4).

Note: Same issue is also repro with "Steak", "Chicken" and "Tofu" buttons under 'Food Order with Validation' card,

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
Hi @anna-dingler and @paulcam206 

Is there any update on this issue as this issue is still repro? As the control is expandable button control then it should announce the state infomration.

**Comment by Rishikesh6521** [comment]
Closing this bug as this bug has been active for more than 2 years logging a same new bug with the latest attachment and tracking the bug through #9241
